CommandHelper/LocalPackages/commands/chars/malice

```diff
@@ -0,0 +1,112 @@
+class: malice
```

Comment:
add .yml to the file name.

Should be:
CommandHelper/LocalPackages/commands/chars/malice.yml

You have:
CommandHelper/LocalPackages/commands/chars/malice